src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -117,4 +125,21 @@
   <ItemGroup>
     <Compile Remove="repro\*" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <TargetOSComponent>unix</TargetOSComponent>
+    <TargetOSComponent Condition="'$(TargetOS)' == 'windows'">win</TargetOSComponent>
+    <TargetSpec>$(TargetOSComponent)_$(TargetArchitecture)_$(TargetArchitecture)</TargetSpec>
+  </PropertyGroup>
+
+  <!-- On Windows we can re-use PGO-optimized clrjit.dll produced by the build for AOT compilation.
```

Comment:
I am not sure whether it is worth the trouble that it may cause (e.g. with symbols), but since you have done it we can give it a try.

Reply:
This logic is copied from crossgen2.csproj.